translations/kubectl/de_DE/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3190 @@
+# German translation.
+# Copyright (C) 2017
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR steffenschmitz@hotmail.de, 2017.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: kubernetes\n"
+"Report-Msgid-Bugs-To: EMAIL\n"
+"POT-Creation-Date: 2017-09-02 01:36+0200\n"
+"PO-Revision-Date: 2017-09-02 01:36+0200\n"
+"Language: de_DE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Last-Translator: Steffen Schmitz <steffenschmitz@hotmail.de>\n"
+"Language-Team: Steffen Schmitz <steffenschmitz@hotmail.de>\n"
+"X-Generator: Poedit 1.8.7.1\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+
+#: pkg/kubectl/cmd/create_clusterrolebinding.go:35
+msgid ""
+"\n"
+"\t\t  # Create a ClusterRoleBinding for user1, user2, and group1 using the "
+"cluster-admin ClusterRole\n"
+"\t\t  kubectl create clusterrolebinding cluster-admin --clusterrole=cluster-"
+"admin --user=user1 --user=user2 --group=group1"
+msgstr ""
+"\n"
+"\t\t # Erstellt ein ClusterRoleBinding für user1, user2 und group1 mit der "
+"cluster-admin ClusterRole\n"
```

Comment:
`mit der ClusterRole cluster-admin``

Otherwise, German requires a dash between the two.